src/content/blog/idx-official-online-editor-partner.mdx

```diff
@@ -0,0 +1,58 @@
+coverImage: "/src/content/blog/_images/idx-official-online-editor-partner/astro-idx-header.webp"
+lang: "en"
+---
+
+import Blockquote from "/src/components/Blockquote.astro"
+import BlogContentImage from "/src/components/BlogContentImage.astro"
+import projectIDXSpace from "/src/content/blog/_images/idx-official-online-editor-partner/project-idx-space.png"
+import logoGoogleOSS from "/src/content/blog/_images/idx-official-online-editor-partner/google-oss-logo.png"
+
+We are excited to announce our new partnership with [Google IDX](https://idx.dev/?utm_source=astro&utm_medium=astro&utm_campaign=astro), who will be joining as Astro’s **Official Online Editor Partner**. IDX is an AI-assisted workspace for full-stack, multi-platform app development in the cloud.
+
+IDX will be sponsoring $10,000 each month towards the ongoing open source maintenance and development of Astro. The new sponsorship funding will be invested directly back into our open source community and long-term maintenance, benefiting everyone.
+
+<BlogContentImage src={projectIDXSpace} alt="Dark graphic depicting outer space and the Project IDX logo." />
+
+## Commitment to Open Source
+
+Google has been committed to open-source software over the last two decades. 
+
+One of the most well-known examples is open-sourcing Chromium. This resulted in improved security, better developer tools, and a focus on web standards. Lesser known may be Google's internal culture. Google employees are incentivized to contribute to open source with the [Open Source Peer Bonus](https://opensource.google/documentation/reference/growing/peer-bonus). Google also created [Google Summer of Code](https://summerofcode.withgoogle.com/) (GSoc). GSoc is a global online program that encourages new contributors to open-source projects.  
+
+David East of IDX shares their thoughts on what this partnership means for both teams:
+
+<Blockquote>
+	<Fragment slot="quote" >
+		I've been a huge Astro fan since the early alpha days, and it's been awesome to see it grow into such a powerful framework. We chose it for idx.dev because we knew it would give us the speed and flexibility we needed, and now, partnering with the Astro team is a fantastic opportunity to give back to that community. Together, we're all about helping developers build the kind of fast, super dynamic, and truly accessible websites that represent the future of the web.
+	</Fragment>
+	<Fragment slot="caption">
+		— [David East](https://x.com/_davideast?lang=en), Lead DevRel at IDX
+	</Fragment>
+</Blockquote>
+
+The last time we partnered with David was to write a case study about Firebase cutting publishing time by building their blog with Astro. We're excited to continue our strong partnership with David, a long-time supporter of Astro who will be leading the partnership from the IDX side.
+
+<BlogContentImage src={logoGoogleOSS} alt="Light graphic depicting Google's OSS logo." />
```

Comment:
Nit: this might be a bit too long of a quote to look good like this? I find it kind of hard to read:

![image](https://github.com/user-attachments/assets/47e472e9-303e-4000-8efc-8eda45f62b46)

Could we treat it pull-quote style and only use the component for a highlight at the end of the section instead of the graphic?

Also suggesting we link to the case study here! Links are goooood!

```suggestion
David East of IDX shares their thoughts on what this partnership means for both teams:

> “I've been a huge Astro fan since the early alpha days, and it's been awesome to see it grow into such a powerful framework. We chose it for idx.dev because we knew it would give us the speed and flexibility we needed, and now, partnering with the Astro team is a fantastic opportunity to give back to that community. Together, we're all about helping developers build the kind of fast, super dynamic, and truly accessible websites that represent the future of the web.”

The last time we partnered with David was to write a case study about [how Firebase cut publishing time by building their blog with Astro](https://astro.build/case-studies/firebase/). We're excited to continue our strong partnership with David, a long-time supporter of Astro who will be leading the partnership from the IDX side.

<Blockquote>
	<Fragment slot="quote" >
		We chose Astro for idx.dev because we knew it would give us the speed and flexibility we needed
	</Fragment>
	<Fragment slot="caption">
		— [David East](https://x.com/_davideast?lang=en), Lead DevRel at IDX
	</Fragment>
</Blockquote>
```